curriculum/challenges/english/20-upcoming-python/learn-regular-expressions-by-building-a-password-generator/6565cbe0a411c37145c55551.md

```diff
@@ -0,0 +1,118 @@
+---
+id: 6565cbe0a411c37145c55551
+title: Step 67
+challengeType: 20
+dashedName: step-67
+---
+
+# --description--
+
+Finally, delete the `count` variable.
```

Comment:
```suggestion
Finally, delete the `count` variable because the `all()` function will return `True` or `False` and you don't need to keep track of the count anymore.
```